plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt

```diff
@@ -300,6 +300,13 @@ app.cash.copper..copper-flow=version.app.cash.copper
 app.cash.copper..copper-rx2=version.app.cash.copper
 app.cash.copper..copper-rx3=version.app.cash.copper
 app.cash.turbine..turbine=version.app.cash.turbine
+co.touchlab..kermit-bugsnag-test=version.kermit-bugsnag-test
+co.touchlab..kermit-bugsnag=version.kermit-bugsnag
+co.touchlab..kermit-crashlytics-test=version.kermit-crashlytics-test
+co.touchlab..kermit-crashlytics=version.kermit-crashlytics
+co.touchlab..kermit-gradle-plugin=version.kermit-gradle-plugin
+co.touchlab..kermit-test=version.kermit-test
+co.touchlab..kermit=version.NO-RULE
```

Comment:
This one doesn't look right. The ones above also don't look right, unless you really have an independent version per artifact.

Reply:
Found out the culprit. The code that reads the `rawRule` of `DependencyGroup`s before putting them in the `dependency-groups-alias-rules.txt` file was a regular tests, which meant it didn't run in time. I made it run `@BeforeAll` instead, so it won't happen again, I'll push in a few minutes.

Reply:
Took me more than a few minutes to push the entire thing (I had WIP dependency notation removals to finish dealing with), but it's there at last!